package.json

```diff
@@ -16,8 +16,9 @@
     "babel": "rimraf lib && babel --stage 0 modules/ -d lib/",
     "dist": "webpack && webpack --config=webpack.config.minified.js",
     "examples": "webpack-dev-server --config examples/webpack.config.js --no-info --content-base examples/",
+    "flow": "node -e \"process.exit(process.platform === 'win32' ? 0 : 1)\" || flow check",
     "lib": "npm run babel && rimraf lib/__tests__ lib/__mocks__",
-    "lint": "eslint modules",
+    "lint": "eslint modules && npm run flow",
```

Comment:
Hmmm, IMO that's asking for people to forget to run it before they commit. CI would catch it down the road, but it would be nice to not have to run yet another command while you're developing. Since eslint (+ other linters) already have a bunch of rules that amount to type checks, they fit in the same category in my mind.

(Not many projects are using `flow-bin`, but at least one of them does have flow in its lint command.)

I can be swayed either way but curious what others think.